build/development-guide/composable-chains/README.md

```diff
@@ -11,55 +11,62 @@ Please contact us in our [Discord tech chat](https://discord.gg/Xb3CxcjCVc) to b
 
-The [XCM Fungible Asset Implementation Guide](https://github.com/open-web3-stack/open-runtime-module-library/discussions/385) has laid out cross-chain fugible asset design considerations and discussions, as well as a reference implementation orml-xtoken that Acala and many others are curerntly adopted and tesitng.
+The [XCM Fungible Asset Implementation Guide](https://github.com/open-web3-stack/open-runtime-module-library/discussions/385) has laid out cross-chain fungible asset design considerations and discussions, as well as a reference implementation orml-xtoken that Acala and many others are currently adopted and testing.
 
 `orml-xtokens` is a reference implementation of XCM for fungible tokens. The source code for xtoken is [here](https://github.com/open-web3-stack/open-runtime-module-library/tree/master/xtokens) and xcm-support is [here](https://github.com/open-web3-stack/open-runtime-module-library/tree/master/xcm-support).
 
+Currently, the `xtoken` codebase is under development by Acala, please reach out to us in [Discord tech chat](https://discord.gg/Xb3CxcjCVc) if you need support.
+
+## Integration Guide
+
+This guide shows how to set up cross-chain transfers between two parachains (**parachain A** and **parachain B**) for sending through their native tokens (**A-Token** and **B-Token** respectively) and **Relay Chain** assets (e.g. DOT in Polkadot).
+
+Transfer messages are wrapped in XCM format and delivered using Horizontal Relay-routed Message Passing (HRMP) channels.
+
+> While [XCMP](https://wiki.polkadot.network/docs/en/learn-crosschain) is still being implemented, a stop-gap protocol (see definition below) known as [HRMP](https://wiki.polkadot.network/docs/en/learn-crosschain#horizontal-relay-routed-message-passing-hrmp) exists in its place. HRMP has the same interface and functionality as XCMP but is much more demanding on resources since it stores all messages in the Relay Chain storage. When XCMP has been implemented, HRMP is planned to be deprecated and phased out in favor of it.
+
 ### Step 0 Local Parachain Testnet
 
-Follow [this guide](https://hackmd.io/dhmCATb_QqygCPxkxaDcmA) by @bertstachios to setup a local parachain testnet environment.
+Follow [this guide](https://hackmd.io/dhmCATb_QqygCPxkxaDcmA) by @bertstachios to set up a local parachain testnet environment.
 
-### Step 1 Support Acala Tokens
+### Step 1 Parachain A should include B-Token (native parachain B token).
```

Comment:
As this guide is for *composable with Acala*, I think we should keep the description as *Acala and your parachain*, instead of *parachain A* and *parachain B*. All the below steps should be updated based on this.

Reply:
@bette7 what do you think?